test/e2e/storage/ubernetes_lite_volumes.go

```diff
@@ -44,7 +44,7 @@ var _ = utils.SIGDescribe("Multi-AZ Cluster Volumes", func() {
 	var err error
 	image := framework.ServeHostnameImage
 	ginkgo.BeforeEach(func() {
-		e2eskipper.SkipUnlessProviderIs("gce", "gke")
+		e2eskipper.SkipUnlessProviderIs("gke")
```

Comment:
I think only the "should only be allowed to provision PDs in zones where nodes exist" test case which creates a vm instance is problematic right?

We actually recently removed a similar test. Maybe this test case should be removed too: https://github.com/kubernetes/kubernetes/pull/103002

cc @TeweiLuo @mattcary

Reply:
I am fine with removing the test altogether but it is storage sig's call. Let me know what needs to be done.

Reply:
I spoke with @mattcary offline and we agree that the "should only be allowed to provision PDs in zones where nodes exist" should be removed for the same reason as #103002

Are you seeing issues with the other test case in your environment?

Reply:
Nope, `should only be allowed to provision PDs in zones where nodes exist"` is the test which is causing problems for us too. I'll remove it.

Reply:
SGTM. thanks.